docs/reference.rst

```diff
@@ -107,14 +107,14 @@ Example configuration:
 ``boot-attempts``
   This configures the number of boot attempts to set when a slot is marked good
   through the D-Bus API or via the command line tool.
-  This is currently only supported when ``bootloader`` is set to ``uboot`` and
-  defaults to 3 if not set.
+  This is currently only supported when ``bootloader`` is set to ``uboot`` or
+  ``barebox`` and defaults to 3 if not set.
```

Comment:
This should mention that the value should match the default configured in the bootloader for consistency.

Reply:
Ack